test/recipe/admin/.baked/rich-transient-with-datas.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$type": "Report",
+  "$type": "Record",
   "path": "rich-transient-with-datas",
```

Comment:
let's make this path `/rich-transient-with-datas` to make it look like a path.

or even better to make it something like `/rich-transient-with-datas/[0]`. and then format it in the record component using `params[1]`
  - logic goes like it's `n+1`; `[0] => params[0 + 1]`